Query/Event/PreFilterConditionCreationEvent.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Query\Event;
+
+use Netdudes\DataSourceryBundle\DataType\DataTypeInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class PreFilterConditionCreationEvent extends Event
+{
+    /**
+     * @var DataTypeInterface
+     */
+    private $dataType;
+
+    /**
+     * @var mixed
+     */
+    private $value;
+
+    /**
+     * @param mixed             $value
+     * @param DataTypeInterface $dataType
+     */
+    public function __construct($value, DataTypeInterface $dataType)
```

Comment:
The `$value` is actually database value. Shouldn't we call it this way for better readability?
And does it make sense to allow users (I mean devs, class users) to modify the value (normal value) as well?